modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtil.java

```diff
@@ -0,0 +1,346 @@
+                                    "format": "uuid"
+                                  }
+                                }
+                              }
+                            },
+                            "400": {
+                              "description": "Bad Request",
+                              "content": {
+                                "*/*": {
+                                  "schema": {
+                                    "$ref": "#/components/schemas/Response"
+                                  }
+                                }
+                              }
+                            },
+                            "404": {
+                              "description": "Not Found",
+                              "content": {
+                                "*/*": {
+                                  "schema": {
+                                    "$ref": "#/components/schemas/Response"
+                                  }
+                                }
+                              }
+                            },
+                            "500": {
+                              "description": "Internal Server Error",
+                              "content": {
+                                "*/*": {
+                                  "schema": {
+                                    "$ref": "#/components/schemas/Response"
+                                  }
+                                }
+                              }
+                            }
+                          }
+                        }
+                      }
+                    },
+                    "components": {
+                      "schemas": {
+                        "Response": {
+                          "required": [
+                            "code",
+                            "message",
+                            "success"
+                          ],
+                          "type": "object",
+                          "properties": {
+                            "code": {
+                              "type": "string",
+                              "description": "The result code of response."
+                            },
+                            "message": {
+                              "type": "string",
+                              "description": "The result message of response."
+                            },
+                            "success": {
+                              "type": "boolean",
+                              "description": "The success boolean of response."
+                            }
+                          }
+                        },
+                        "property": {
+                          "required": %s,
+                          "type": "object",
+                          "properties": %s
+                        },
+                        "CreateRequest": {
+                          "required": [
```

Comment:
can this list be also read dynamically?

Reply:
Yes. I will add a method to get the list of fields in the Class 'CreateRequest' which has @NotNull.